base/special/cbrt.jl

```diff
@@ -0,0 +1,121 @@
+# ====================================================
+# Copyright (C) 1993 by Sun Microsystems, Inc. All rights reserved.
+#
+# Developed at SunPro, a Sun Microsystems, Inc. business.
+# Permission to use, copy, modify, and distribute this
+# software is freely granted, provided that this notice
+# is preserved.
+# ====================================================
+#
+# Optimized by Bruce D. Evans.
+
+# s_cbrtf.c -- float version of s_cbrt.c.
+# Conversion to float by Ian Lance Taylor, Cygnus Support, ian@cygnus.com.
+# Debugged and optimized by Bruce D. Evans.
+
+cbrt(x::Real) = cbrt(float(x))
+
+issubnormal(::Float64, hw) = hw < 0x00100000
+issubnormal(::Float32, hw) = hw < 0x00800000
```

Comment:
Base already exports an `issubnormal` that this should use.

Reply:
I've used it. It does cost 0.3 ns more because I lumped 0 and subnormals before :)